lms/djangoapps/ci_program/models.py

```diff
@@ -73,8 +73,12 @@ class Meta:
     video = models.URLField(null=True, blank=True)
     program_code = models.CharField(max_length=50, null=True, blank=True)
     specialization_for = models.CharField(max_length=50, null=True, blank=True)
-    sample_content_for = models.CharField(max_length=50, null=True, blank=True)
-    support_program_for = models.CharField(max_length=50, null=True, blank=True)
+    sample_content = models.CharField(max_length=50, null=True, blank=True)
```

Comment:
Have we tested migrating this on staging - specifically, is there a window between deploying EDX and running the migration where EDX is down?

Reply:
The migration on Staging was apparently performed automatically (?), I never had to run migrate.

Reply:
yeah that might've been too clever :) probably worth checking in the morning, which might be awkward and delay the release sorry...